frontend/src/components/dashboard/myJobCents.jsx

```diff
@@ -89,13 +101,15 @@ export default class MyJobCents extends React.Component {
                     this.props.goToChallengeDetail(sponsoredChallenge, sponsoredChallengeBalances[index], sponsoredChallengeRemainingRedemptions[index])
                 }}>
                     <h2 className="balance-title">{sponsoredChallenge.company}</h2>
-                    <img className="logoImg" src={sponsoredChallenge.imageUrl || ncentLogo} alt="ncent logo"/>
+                    <img className="logoImg" src={sponsoredChallenge.imageUrl || ncentLogo} alt="ncent logo" 
+                        onError={this.imgLoadError} onLoad={this.imgLoad}
+                    />
                     <h2 className="balance-title">{sponsoredChallenge.name}</h2>
                     <h3 className="balance-subtitle">Sponsored</h3>
-                    <h3 className="balance-subtitle">{sponsoredChallengeBalances[index]} jobCent(s)</h3>
+                    <h3 className="balance-subtitle">{ sponsoredChallengeBalances[index]  === 1 ? sponsoredChallengeBalances[index] + " jobCent" : sponsoredChallengeBalances[index] + " jobCents"}</h3>
                 </div>
                 <a
-                    title="New"
+                    title="Send jobCents to another user"
```

Comment:
is this too verbose?

Reply:
I thought it was a bit more user-friendly.  The old title/tooltip was just "New" and it didn't really explain what the tab was doing.